test/smoke/quorum.js

```diff
@@ -0,0 +1,181 @@
+const createRpcClientFromConfig = require('../../lib/test/createRpcClientFromConfig');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const { inventory, network, variables } = getNetworkConfig();
+
+const allHosts = inventory.masternodes.hosts.concat(
+  inventory.wallet_nodes.hosts,
+  inventory.miners.hosts,
+  inventory.seed_nodes.hosts,
+);
+
+const quorumCheckTypes = {
```

Comment:
I guess the list should depend on the network, isn't? Or testnet and devnets have the same set?

Reply:
Yeah, I'm not really sure how to configure this properly. dkgInterval is also the wrong measurement, quorums seem to get older than 24 blocks. How does this work?

Reply:
You probably need to have a map of which quorum types and the number of quorums should exist for testnet and devnet. Then use https://github.com/dashpay/dashcore-lib/blob/master/lib/deterministicmnlist/QuorumEntry.js#L335 to get params for specific quorum type.

Reply:
How should I use this? It looks like it returns useful vars about the number of quorum members, should we add a check for that? Is there a similar function I can use to get the quorum lifetimes, or is it fine to use `dkgInterval * 1.5`?

Reply:
>  It looks like it returns useful vars about the number of quorum members, should we add a check for that?
I don't think so. If we have quorum in list it's already has min number of valid members.
>  Is there a similar function I can use to get the quorum lifetimes
I don't think so